debian/po/templates.pot

```diff
@@ -0,0 +1,57 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
this is a template, why check it in?

Reply:
Wholesale copy. Can clean up later. I'm still not sure what some of these do, perhaps they need to exist or maybe not. Would need to test.

Reply:
Should probably be changed from `jellyfin-server` to `jellyfin-web`, but no need to do that in this PR.